brave/browser-laptop
# Update siteInfo.js to the modified BEM style

Closes #10932

Auditors:

Test Plan:
1. Open https://mixed-script.badssl.com/
2. Open https://mixed.badssl.com/ in a new tab
3. Open http://http.badssl.com/ in a new tab
4. Click the lock icon of each tab
5. Make sure the site info dialogs are properly displayed

Submitter Checklist:


Test Plan:


Reviewer Checklist:

Tests
